keras_cv/backend/ops.py

```diff
@@ -25,5 +28,5 @@
         from keras_core.src.utils.image_utils import (  # noqa: F403, F401
             smart_resize,
         )
-else:
+if not multi_backend():
     from keras_cv.backend.tf_ops import *  # noqa: F403, F401
```

Comment:
Why are we importing `tf_ops` twice? Once here and once from the toplevel `__init__.py`? Do we need both?

Reply:
I think this was added to have tf_ops to be available through `keras_cv.backend.ops.*` instead if `keras_cv.backend.tf_ops.*`